staff/oriol-escolar/cv/index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV Oriol Escolar</title>
+</head>
+
+<body>
+    
+
+<h1>Oriol Escolar</h1>
+<img src="" alt="Image sample">
+
+<!-- EDUCATION -->
+<section>
+ 
+<h2><u>Education</u></h2>
```

Comment:
The <u> element was deprecated in HTML 4.01. (the <u> element was used to define underlined text).

Reply:
You are right, removed <u> in the code.